.github/workflows/pythonapp.yml

```
@@ -237,3 +237,39 @@ jobs:
           pytest -v
           test/test_cvedb.py
           test/test_cli.py
+
+  cve_scan:
+    name: CVE Scan on dependencies
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Get Date
+        id: get-date
+        run: |
+          echo "::set-output name=date::$(/bin/date -u "+%Y%m%d")"
+        shell: bash
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: get cached python packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: get cached database
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/cve-bin-tool
+          key: ${{ runner.os }}-cve-bin-tool-${{ steps.get-date.outputs.date }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -r doc/requirements.txt
```

These can be combined into one command:
```suggestion
          python -m pip install -r requirements.txt -r doc/requirements.txt
```

---

The commands are seperate in other jobs. I guess I will leave it like that here.

---

Hm, where are they separate? We only install from both reqs in one other job and it is combined there:
https://github.com/intel/cve-bin-tool/blob/967a03edcd0a938196cc04482f524909d0b3ab26/.github/workflows/pythonapp.yml#L34